### ISSUE TITLE
Error exit code when using --list

### ISSUE THREAD

**Issue by belgattitude** [ocavue/pnpm-deduplicate#4]
First of all thanks for this script.

I'd like to have an error exit code to add a check on the ci

For example when running: `pnpm exec pnpm-deduplicate --list`

![image](https://user-images.githubusercontent.com/259798/194947416-5df1219b-a92f-494b-a17a-06bb05d2eb63.png)

Rather than zero have an error code = 1 (for example), That way it's easy to add a github action

```yaml
      - name: 🔎 Check for duplicates
        run: |
          pnpm exec pnpm-deduplicate --list
```

**Comment by ocavue** [comment]
Good idea.

**Comment by ocavue** [comment]
Shipped in version 0.4.0.
